app/models/peep.rb

```diff
@@ -0,0 +1,11 @@
+class Peep
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :message, String, required: true
+  property :time, DateTime
+
+  validates_presence_of :message
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.